CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
](https://www.github.com/googleapis/python-translate/issues/659) [#715](https://www.github.com/googleapis/python-translate/issues/715) [#749](https://www.github.com/googleapis/python-translate/issues/749) [#802](https://www.github.com/googleapis/python-translate/issues/802) [#843](https://www.github.com/googleapis/python-translate/issues/843) [#886](https://www.github.com/googleapis/python-translate/issues/886) [#887](https://www.github.com/googleapis/python-translate/issues/887) [#910](https://www.github.com/googleapis/python-translate/issues/910) [#962](https://www.github.com/googleapis/python-translate/issues/962) [#1004](https://www.github.com/googleapis/python-translate/issues/1004) [#1055](https://www.github.com/googleapis/python-translate/issues/1055) [#1093](https://www.github.com/googleapis/python-translate/issues/1093) [#1121](https://www.github.com/googleapis/python-translate/issues/1121) [#1158](https://www.github.com/googleapis/python-translate/issues/1158) [#1186](https://www.github.com/googleapis/python-translate/issues/1186) [#1192](https://www.github.com/googleapis/python-translate/issues/1192) [#1254](https://www.github.com/googleapis/python-translate/issues/1254) [#1377](https://www.github.com/googleapis/python-translate/issues/1377) [#1441](https://www.github.com/googleapis/python-translate/issues/1441) [#1464](https://www.github.com/googleapis/python-translate/issues/1464) [#1488](https://www.github.com/googleapis/python-translate/issues/1488) [#1787](https://www.github.com/googleapis/python-translate/issues/1787) [#1980](https://www.github.com/googleapis/python-translate/issues/1980) [#2084](https://www.github.com/googleapis/python-translate/issues/2084) [#2323](https://www.github.com/googleapis/python-translate/issues/2323) [#2327](https://www.github.com/googleapis/python-translate/issues/2327) [#2325](https://www.github.com/googleapis/python-translate/issues/2325) [#2379](https://www.github.com/googleapis/python-translate/issues/2379) [#2400](https://www.github.com/googleapis/python-translate/issues/2400) [#2439](https://www.github.com/googleapis/python-translate/issues/2439) [#2498](https://www.github.com/googleapis/python-translate/issues/2498) [#2671](https://www.github.com/googleapis/python-translate/issues/2671) [#2677](https://www.github.com/googleapis/python-translate/issues/2677) [#2731](https://www.github.com/googleapis/python-translate/issues/2731) [#2759](https://www.github.com/googleapis/python-translate/issues/2759) [#2745](https://www.github.com/googleapis/python-translate/issues/2745) [#2768](https://www.github.com/googleapis/python-translate/issues/2768) [#2769](https://www.github.com/googleapis/python-translate/issues/2769) [#2937](https://www.github.com/googleapis/python-translate/issues/2937) [#2914](https://www.github.com/googleapis/python-translate/issues/2914) [#3046](https://www.github.com/googleapis/python-translate/issues/3046) [#3084](https://www.github.com/googleapis/python-translate/issues/3084)
```

Comment:
@busunkim96 I'm guessing these issues are being tagged because the sample migration commits were included in my library migration PR. OK if I just delete these from the CHANGELOG?

Reply:
@danoscarmike Yes, please delete them. :)